### ISSUE TITLE
/tmp/docker-certfile flooding

### ISSUE THREAD

**Issue by hxpro** [stage1/docker-php#145]
Each run of my script create new file in /tmp folder with my cert and private key that i use to connect to docker api. If omitt security issue, there is problem with running out of inodes.

My script can by pretty simple:
``` php
$client = DockerClient::createWithEnv();
$docker = new Docker($client);
```
DOCKER_TLS_VERIFY = 1
DOCKER_PEER_NAME = "docker.myserver.dev"
DOCKER_HOST = "tcp://192.168.1.1:2376"
DOCKER_CERT_PATH = /etc/myapp/

**Comment by hxpro** [comment]
I sent pull request to fix this issue, but testKill do not pass. When run tests on my localhost before changes, same failures assert me.

**Comment by joelwurtz** [comment]
Thanks for the report and the PR, i learned a new option in php :/

Since there was a refactoring on the library i take the liberty to adapt your changes to the new version, feel free to add an `@author` tag in the docker client with a new PR for contribution reference.

closed in #147
